packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -122,29 +122,53 @@ const headlineTypography = script => ({
   leading: headlineLeadingStoryTypography(script),
 });
 
-export const Headline = styled.h3`
-  color: ${C_EBON};
-  margin: 0; /* Reset */
-  padding-bottom: ${GEL_SPACING};
-  ${({ service }) => getSerifMedium(service)}
-  ${({ script, promoType }) => script && headlineTypography(script)[promoType]}
-  ${({ promoHasImage }) =>
-    !promoHasImage &&
-    `display: inline;`} /* Needed for aligning Media Indicator with Headline */
-`;
+export const Headline = ({
+  children,
+  promoHasImage,
+  promoType,
+  script,
+  service,
+  tag,
+}) => {
+  return (
+    <StyledHeadline
```

Comment:
Instead of making this change here, can we update the usage in the Story Promo container:
https://github.com/bbc/simorgh/blob/latest/src/app/containers/StoryPromo/index.jsx#L138-L143
I think that would be a better place to have the logic.

Here we are able to have the standard version and override it with an 'as' prop only when necessary. 
```
import { Headline } from '@bbc/psammead-story-promo';
....
const headlineProps = {
  script,
  service,
  promoType,
  promoHasImage: displayImage,
  ...(headlingTagOverride && { as: headingTagOverride })
};
...
<Headline {...headlineProps}>{...elements in headline here...}</Headline>
```
These Psammead components should have as little conditional rendering as possible.